website/static/js/nodeControl.js

```diff
@@ -265,6 +274,21 @@ function NodeControl (selector, data, options) {
 NodeControl.prototype.init = function() {
     var self = this;
     osfHelpers.applyBindings(self.viewModel, this.selector);
+    if (self.data.user.is_admin && !self.data.node.is_retracted) {
+        new NodesPrivacy('#nodesPrivacy', self.data.node, function(nodesChanged, requestedEmbargoTermination) {
+            // TODO: The goal here is to update the UI of the project dashboard to
+            // reflect the new privacy state(s). Unfortunately, since the components
+            // view is rendered server-side we have a relatively limited capacity to
+            // update the page. Once the components list is componentized we can
+            // rerender just that section of the DOM (as well as updating the
+            // project-specific UI).
+            // For now, this method only needs to handle requests for making embargoed
+            // nodes public.
+            if (requestedEmbargoTermination) {
```

Comment:
The KO view-model scope disallows programmatically changing the state of the Private/Public button. Rather than using $ or equivalent to do this, I chose to pass a callback into NodePrivacy.

From an architecture standpoint, it seems to me that since NodesPrivacy is a subcomponent of NodeControl the NodeControl should handle modifications to the Node and update the state of the child view model as appropriate.